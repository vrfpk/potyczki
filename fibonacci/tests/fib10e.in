023195724199731042
