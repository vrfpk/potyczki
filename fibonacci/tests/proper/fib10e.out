1500000000000010863
